### ISSUE TITLE
Access to the pre-trained model

### ISSUE THREAD

**Issue by roimulia2** [House-Leo/RWSR-EDL#3]
Hey, I loved the paper! 

Two questions:
- Can we have access to the pre-trained model?
- What is the estimated time, in your opinion, to run the model on a single image on a **mobile device**?
- What is the recommended image resolution input size? 

Thanks!

**Comment by House-Leo** [comment]
Thanks for your attention!

- Since I have graduated from Guangdong University of Technology, I can not connect to the GPU server now. But I think you can easily get the similar results by following README.
- The estimated time in Tab. VI is calculated on the GPU server with Tesla V100 not a mobile device. The resolution of the input image is 300x200 with 4x SR. By the way, I do not think RealSR can achieve satisfactory results with an efficient model. But if you have done some research on this, I'd be interested in that too :)
- You can get the training image patch size in option folder. About the testing image, the resolution of most of the images in the DIV2K val/test set is 510x340.

Hope the above can answer your questions.

**Comment by roimulia2** [comment]
Thank you so much!
